bundle/tests/loader.go

```diff
@@ -15,29 +15,33 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func load(t *testing.T, path string) *bundle.Bundle {
+func load(t *testing.T, path string, mutators ...bundle.Mutator) *bundle.Bundle {
```

Comment:
Better keep `load` as-is and just do `bundle.Apply` separately for tests which needs them

Reply:
This would require refactoring load to accept the context. This was my original idea but it is used in many places so I skipped that.

Reply:
It's fine to use a new context, if you need something from the context you can set it up in the test itself so it's transparent